doc/VTGateV3Features.md

```diff
@@ -0,0 +1,86 @@
+# VTGate V3 features
+
+## Overview
```

Comment:
As very first sentence, I would like to see here a definition of vtgate first. This will give a better introduction.

For me, vtgate is a proxy which contains the routing logic. You can also mention here that it's basically a fat client, but provided as a service. Over time, we moved away from a fat client, which had to be incorporated into the application, to the proxy model. This way, the routing logic is more decoupled from the application and easier to maintain separately.